README.md

```diff
@@ -1,24 +1,23 @@
 # edgex-snap-hooks
-[![Go Reference](https://pkg.go.dev/badge/github.com/canonical/edgex-snap-hooks.svg)](https://pkg.go.dev/github.com/canonical/edgex-snap-hooks)
+[![Go Reference](https://pkg.go.dev/badge/github.com/canonical/edgex-snap-hooks.svg)](https://pkg.go.dev/github.com/canonical/edgex-snap-hooks/v2)
 
-Snap hooks library use by Go EdgeX service snaps.  
-
-### What is this repository for? ###
-
-This module provides support for Go-based configure an install hooks...
+Snap hooks library used by Go EdgeX service snaps.  
+It provides utilites to implement snap hooks, including some wrappers for `snapctl` commands.
 
 ### Installation ###
+Make sure you have modules enabled, i.e. have an initialized `go.mod` file.
 
-* Make sure you have modules enabled, i.e. have an initialized  go.mod file
-* If your code is in your GOPATH then make sure ```GO111MODULE=on``` is set
-* Run ```go get github.com/canonical/edgex-snap-hooks```
-  * This will add the edgex-snap-hooks to the go.mod file and download it into the module cache
+Run:
+```
+go get github.com/canonical/edgex-snap-hooks/v2
```

Comment:
Would `go mod tidy` not be a simpler way of doing this as we then don't need to specify the version? It does however do a number of other cleanup tasks as well, so I'm fine with it if you want to leave the instructions at this, using `go get`

Reply:
Are you suggesting to manually add the package inside go.mod and run go mod tidy? This would also need the version.

Or import the package in the code and then run go mod tidy? This requires writing some code (without any IDE helpers, since the package doesn't exist yet) that uses the import or doing an anonymous import (with the blank _ identifier), and then go mod tidy.

`go get github.com/canonical/edgex-snap-hooks/v2@vX.X.X` can also do the upgrade..

Reply:
We can add a reference to https://pkg.go.dev/cmd/go#hdr-Add_dependencies_to_current_module_and_install_them

Reply:
ok, that's fair enough. I assumed the user would have created the package with the import statement already. It depends on the workflow I suppose. In that case, just leave this as it is - with "go get" - that makes sense.